src/Components/ModelledPropertyBuilder/ModelledPropertyBuilder.styles.ts

```diff
@@ -100,3 +102,18 @@ export const propertyExpressionLabelStyles: Partial<ILabelStyles> = {
         paddingTop: 5
     }
 };
+
+export const getActionButtonStyles = memoizeFunction(
```

Comment:
it seems like it should be right aligned according to the designs. 
![image](https://user-images.githubusercontent.com/57726991/171518671-14f298f0-8e1c-4b94-ba56-48496475b9c9.png)

Actual
![image](https://user-images.githubusercontent.com/57726991/171518780-2e2d93d2-bbc1-44e9-8ae1-850a15bd62a0.png)

Reply:
Design link - https://www.figma.com/file/LsmZeVOZJgwZs75HPTQmgU/3D-Scenes-Mock-Screens?node-id=45549%3A254023

Reply:
yeah also def no italics IMO